### PR TITLE
Steam, Twitch and Spotify added

### PR DESCRIPTION
**Pull request by Francismori7**
Fixes three issues!

### CODE REVIEW

**Comment by Francismori7** [comment]
And sorry about the indentation, Visual Studio found it funny to rearrange the document -_-

**Comment by lipis** [comment]
I am sorry.. but I think it is very clear here: https://github.com/lipis/bootstrap-social/blob/gh-pages/.editorconfig#7

**Comment by Francismori7** [comment]
Let me see what I can do. Can I just commit over those? :|

**Comment by lipis** [comment]
Actually I'm not sure if I will add all these icons anyway.. as like I mentioned before there are already too many and it's super easy to add new ones if using Less/SASS :) But when folks are getting only the CSS is kind of annoying to get a massive file with 95% useless stuff..

If I'll break down the core icons and the rest then maybe..

File: bootstrap-social.less [hunk 1]
```diff
@@ -18,7 +18,10 @@
   text-align: left;
   white-space: nowrap;
   overflow: hidden;
+  -ms-text-overflow: ellipsis;
+  -o-text-overflow: ellipsis;
```
Comment (lipis) [inline bootstrap-social.less:22]: http://caniuse.com/#feat=text-overflow

**Comment by Francismori7** [comment]
Your call, just trying to help out :+1:

**Comment by lipis** [comment]
Yes no problem and I can see the need of adding new icons...

But looking forward.. why are you changing the CSS and adding new rules.. and all that stuff when this PR is for adding new icons..? if there is something that needs to be changed make a separate one..

**Comment by lipis** [comment]
```
505 additions and 324 deletions
```

For 3 new icons is just doesn't make any sense :) sorry

File: bootstrap-social.less (hunk 1 above)
Comment (Francismori7) [inline bootstrap-social.less:22]: Once again, Web Essentials playing with my nerves -_-

**Comment by Francismori7** [comment]
Yeah about all those additions and stuff, the whole reformating of the files created a massive clutter in changing, most of them are new-lines and line identation changes. Nothing else has been changed (if we consider the text-overflow to be an issue on my end)

**Comment by topless** [comment]
Since the change is small, I would suggest, to delete that branch and go through the trouble to recreate the pr with the suggestions from @lipis in a lighter editor like atom or a properly configured VS.

**Comment by Francismori7** [comment]
Why go through the trouble considering @lipis is not certain to merge it in?

**Pull request closed without merging**